CONTRIBUTING.md

```diff
@@ -63,13 +63,19 @@ Our commit style is defined in a __[commit template]__. Use it as a reference or
 
 ### Rules and Recommendations
 
-- *Use your __real name__ and __real email__.* We do not accept anonymous contributions!
+- *Use your __real name__ and __real email__.* We do not accept anonymous code contributions!
+  - Every commit that changes the code or translations should have author's full legal name (in latin letters, diacritics allowed).
+  - Same applies to the author's GitHub profile, it should have full name set (in the [Name field here][GitHub Profile Settings]) that matches one specified in the commits.
```

Comment:
Why do we want to enforce people to fill in their github profile?

Reply:
If I understand correctly, it's to be able to press the "Squash & Merge" button in their PRs (from the github webinterface) so as to get their name OK. Instead to do the whole work of checking their PR locally and manually do the adjustments.

Reply:
Yes, for the same reason the author's GitHub name display has been added into the [tampermonkey script](https://github.com/learn-more/tampermonkey) for ReactOS PRs.

Reply:
I am not in favor of forcing people to set that.